boards/posix/native_posix/doc/board.rst

```diff
@@ -159,7 +159,16 @@ Run the zephyr.exe executable as you would any other Linux console application.
 This executable accepts several command line options depending on the
 compilation configuration.
 You can run it with the ``--help`` command line switch to get a list of
-available options.
+available options::
+
+   $ zephyr/zephyr.exe --help
+
+     [-h] [--h] [--help] [-?]  :Display this help
+     [-stop_at=<time>]         :In simulated seconds, when to stop automatically
+     [-seed=<r_seed>]          :Seed for the entropy device
```

Comment:
2 comments:
* The seed option only appears if the native entropy driver is compiled in.
* This option is now described as (After you requested it to be clarified :) )
 [-seed=<r_seed>]             :A 32-bit integer seed value for the entropy device, such as 97229 (de
                              cimal), 0x17BCD (hex), or 0275715 (octal)